packages/gatsby-theme-docs/src/components/child-sections-nav.js

```diff
@@ -123,15 +124,22 @@ const SectionNavigation = (props) => {
   const item = props.tocNode;
   return (
     <>
-      <Link href={item.url}>{item.title}</Link>
+      {props.isSubItem ? (
+        <Link href={item.url}>
+          <SpacingsInline scale="xs">
+            <div>-</div>
+            <div>{item.title}</div>
```

Comment:
By meaning, this are rather `<span>`s - segements inside the text flow and not block level layout elements.